### PR TITLE
Refactor onboarding user vars to be absent when user is fully onboarded

### PR DESCRIPTION
**Pull request by charlesBochet**
In this PR:
- take feedbacks from: https://github.com/twentyhq/twenty/pull/6530 / https://github.com/twentyhq/twenty/pull/6529 / https://github.com/twentyhq/twenty/pull/6526 / https://github.com/twentyhq/twenty/pull/6512
- refactor onboarding uservars to be absent when the user is fully onboarded: isStepComplete ==> isStepIncomplete
- introduce a new workspace.activationStatus: CREATION_ONGOING

I'm retesting the whole flow:
- with/without BILLING
- sign in with/without SSO
- sign up with/without SSO
- another workspaceMembers join the team
- subscriptionCanceled
- access to billingPortal

### CODE REVIEW

**Review by Weiko — APPROVED** [review]

File: packages/twenty-front/src/modules/prefetch/components/PrefetchRunQueriesEffect.tsx [hunk 1]
```diff
@@ -24,11 +24,20 @@ export const PrefetchRunQueriesEffect = () => {
       prefetchKey: PrefetchKey.AllFavorites,
     });
 
+  const { objectMetadataItems } = useObjectMetadataItems();
```
Comment (charlesBochet) [inline packages/twenty-front/src/modules/prefetch/components/PrefetchRunQueriesEffect.tsx:27]: refactor PrefetchQueries to make sure that they are leveraging operationSignatures

File: packages/twenty-server/src/engine/core-modules/onboarding/onboarding.service.ts [hunk 2]
```diff
@@ -116,29 +90,48 @@ export class OnboardingService {
...
       workspaceId,
-      key: OnboardingStepKeys.ONBOARDING_INVITE_TEAM_COMPLETE,
-      value,
+      key: OnboardingStepKeys.ONBOARDING_INVITE_TEAM_PENDING,
+      value: true,
     });
   }
 
-  async toggleOnboardingCreateProfileCompletion({
+  async setOnboardingCreateProfileCompletion({
```
Comment (martmull) [inline packages/twenty-server/src/engine/core-modules/onboarding/onboarding.service.ts:134]: `setOnboardingCreateProfilePending`

File: packages/twenty-server/src/engine/core-modules/auth/services/sign-in-up.service.ts [hunk 3]
```diff
@@ -175,6 +175,18 @@ export class SignInUpService {
     await this.userWorkspaceService.create(user.id, workspace.id);
     await this.userWorkspaceService.createWorkspaceMember(workspace.id, user);
 
+    await this.onboardingService.setOnboardingConnectAccountPending({
+      userId: user.id,
+      workspaceId: workspace.id,
+      value: true,
+    });
+
+    await this.onboardingService.setOnboardingCreateProfileCompletion({
```
Comment (martmull) [inline packages/twenty-server/src/engine/core-modules/auth/services/sign-in-up.service.ts:184]: should not be called if `firstName!=='' && lastName!==''`

File: packages/twenty-server/src/engine/core-modules/auth/services/sign-in-up.service.ts [hunk 4]
```diff
@@ -222,13 +234,22 @@ export class SignInUpService {
...
-        value: true,
-      });
-    }
+    await this.onboardingService.setOnboardingConnectAccountPending({
+      userId: user.id,
+      workspaceId: workspace.id,
+      value: true,
+    });
+
+    await this.onboardingService.setOnboardingCreateProfileCompletion({
```
Comment (martmull) [inline packages/twenty-server/src/engine/core-modules/auth/services/sign-in-up.service.ts:243]: should not be called if `firstName!=='' && lastName!==''`

**Review by martmull — COMMENTED** [review]
small comments, opening a PR to pass them

**Pull request merged**